cakefuzzer/scanners/utils.py

```diff
@@ -80,3 +85,24 @@ def get_vulnerability_objects(self, **kwargs) -> List[Vulnerability]:
                 f.write(self.string)
 
         return vulnerabilities
+
+
+def find_html_location(
+        contents : str,
+        phrase : str,
+        ) -> None:
+    
+    soup = BeautifulSoup(contents, 'html.parser')
+    tags = []
+    for tag in soup.find_all():
+        if phrase in tag.text:
+            tags.append(f"{tag.name}.text")
+        if phrase.lower() in tag.name:
+            # !!! HTML sees "<_" as "&lt;" so it doesnt recognize it as a tag
+            tags.append("tag")
+        for attr, value in tag.attrs.items():
+            if phrase in value:
+                tags.append(f"{tag.name}.{attr}.value")
+            if phrase.lower() in attr:
+                tags.append(f"{tag.name}.attr")
+    return tags
```

Comment:
Question: is it actually possible that this will ever return more than one element in the list? What would the scenario look like ?